### PR TITLE
change offerId to offer_id

### PR DESCRIPTION
**Pull request by GauravBains**
- Updated Checkout to use offer_id instead of offerId for keeping naming consistent

### CODE REVIEW

**Comment by mm53bar** [comment]
I'm confused here. What consistent naming are you talking about? On the line above is this:

`_config.checkoutFormType = _checkoutElem.getAttribute('checkoutFormType')`

So it seems like camelCase is the naming standard, not snake_case? Or is it a different naming standard you're talking about?

**Comment by hanmindable** [comment]
@mm53bar You've exposed an issue that fell through the cracks - we're not too thrilled with how these "data" fields are defined. I think this is a good time to address Issue #3, which gets us on the HTML5 `data-` convention and create in turn defines our convention.

I consciously decided to defer this in the name of speed, but in hindsight that wasn't a good decision.

**Comment by mm53bar** [comment]
@hanmindable OK sounds good. I'll keep my mouth shut for now because it's all in a state of change. 😄

File: checkout-form-manager.js [hunk 1]
```diff
@@ -621,7 +628,7 @@ function submitCheckout(e) {
 
     const formData = new FormData(form);
     formData.append('hash', (new URL(document.location)).searchParams.get('hash'));
-    formData.append('offer_id', checkoutElement.getAttribute('offerId'));
+    formData.append('offer_id', checkoutElement.getAttribute('offer_id'));
```
Comment (hanmindable) [inline checkout-form-manager.js:631]: This is an oops on my part from before - we should be getting the `offer_id` from the `config`.
```suggestion
    formData.append('offer_id', config['offer_id']);
```

**Review by GauravBains — COMMENTED** [review]
changes look good, once we update instapage we should remove the fallback for config to use HTML attribute instead of data-attributes

**Review by hanmindable — APPROVED** [review]

**Pull request merged**